gallery/package.json

```diff
@@ -14,9 +14,12 @@
     "webpack": "webpack --env production --mode=production",
     "webpack-dev-server": "webpack serve",
     "test": "npm-run-all check-install lint-tests build jest",
-    "ci-test": "npm-run-all check-install ci-lint-tests build jest",
+    "test-dark": "npm-run-all check-install lint-tests build jest-dark",
+    "ci-test": "npm-run-all check-install ci-lint-tests build ci-jest",
     "check-install": "yarn install --check-files --frozen-lockfile",
+    "ci-jest": "jest --ci --testTimeout=120000",
     "jest": "jest --testTimeout=120000",
+    "jest-dark": "RSC_GALLERY_THEME=DARK jest --testTimeout=120000",
```

Comment:
I believe we need a `ci-jest-dark`, and `ci-test` needs to run both of them.  `test` should probably also run both `jest` and `jest-dark` rather than having a separate `test-dark`.

Reply:
> I believe we need a `ci-jest-dark`, and `ci-test` needs to run both of them. `test` should probably also run both `jest` and `jest-dark` rather than having a separate `test-dark`.

I had taken a look at trying to get `jest` and` jest-dark` both to run under a single `test` command, but if there are any snapshots that fail in the initial `jest` command, yarn exits with `"error Command failed with exit code 1"` and `jest-dark` is not tested. Is there a way to override that exit command in this instance where multiple jest commands are run in one `test` command?

Reply:
> I had taken a look at trying to get jest and jest-dark both to run under a single test command, but if there are any snapshots that fail in the initial jest command, yarn exits with "error Command failed with exit code 1" and jest-dark is not tested. Is there a way to override that exit command in this instance where multiple jest commands are run in one test command?

That's not particularly a problem.  All of our build is set up so that a failure at one step prevents later steps from executing.  In some ways it could be advantageous to try to run later steps after earlier failures, but that's quite a bit harder to set up and has some disadvantages too, such as the reason for a failure not necessarily being near the bottom of the build log and thus being harder to find.

Reply:
> That's not particularly a problem. All of our build is set up so that a failure at one step prevents later steps from executing. In some ways it could be advantageous to try to run later steps after earlier failures, but that's quite a bit harder to set up and has some disadvantages too, such as the reason for a failure not necessarily being near the bottom of the build log and thus being harder to find.
Ok that's good to know. So I removed the unnecessary `test-dark` script and added in `ci-jest-dark` here: [7b218cc](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/7b218cce14b47564c98ebeeaa0c24b5e1b0fc58f)